homeassistant/components/environment_canada/camera.py

```diff
@@ -52,18 +52,21 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             coordinates=(lat, lon), precip_type=config.get(CONF_PRECIP_TYPE)
         )
 
-    add_devices([ECCamera(radar_object, config.get(CONF_NAME))], True)
+    add_devices(
+        [ECCamera(radar_object, config.get(CONF_NAME), config.get(CONF_LOOP))], True
```

Comment:
Please use `dict[key]` for required schema keys and keys with default schema values.
```suggestion
        [ECCamera(radar_object, config.get(CONF_NAME), config[CONF_LOOP])], True
```

Reply:
Sure, it's been changed.